src/framework/components/collision/component.js

```diff
@@ -31,8 +37,12 @@ import { Component } from '../component.js';
  * - "sphere": A sphere-shaped collision volume.
  *
  * Defaults to "box".
- * @property {import('../../../core/math/vec3.js').Vec3} halfExtents The half-extents of the
+ * @property {Vec3} halfExtents The half-extents of the
  * box-shaped collision volume in the x, y and z axes. Defaults to [0.5, 0.5, 0.5].
+ * @property {Vec3} linearOffset The positional offset of the collision from the Entity position in local space.
```

Comment:
is it an 'offset of the collision' or 'offset of a collision shape' / volume or similar?

Reply:
Shape makes more sense here